course-definition.yml

```diff
@@ -168,60 +168,61 @@ stages:
     name: "Parse Correlation ID"
     difficulty: medium
     description_md: |-
-      In this stage, you'll parse the correlation ID from a request and send a response with the same correlation ID.
+      In this stage, you'll replace the hard-coded correlation ID with the actual correlation ID from the request.
 
-      ### Request header format
+      ### Request message
 
-      In the previous stage, we saw that the request starts with a 4 byte length field, followed by the request header, and then the request body.
+      A request message has three parts:
+      1. Message size
+      2. Header
+      3. Body
 
-      The format of the request header is as follows:
+      To get the correlation ID, you need to find its offset. You already know that the message size is 4 bytes long. And here's what the request header looks like (in this challenge, we're using [request header v2](https://kafka.apache.org/protocol.html#protocol_messages)):
```

Comment:
Should this also be changed to "In this stage," as per [this comment](https://linear.app/codecrafters/document/kafka-stage-2-ab182594c97e#comment-01c552eb)?

Reply:
Yep!

Reply:
@rohitpaulk done!